src/components/DatePicker/RangePicker.jsx

```diff
@@ -97,16 +122,50 @@ export const RangePicker = ({
   const handleCanChange = pos => value =>
     currentMonth.clone().add(pos, 'month').isSame(moment.utc(value), 'month')
 
+  const computeSecondInterval = ({ start, end }, compareMethod) => {
+    const _start = moment(start)
+    const _end = moment(end)
+
+    const diff = _end.diff(_start, 'days') + 1
+
+    return {
+      start: _start.subtract(diff, 'days').format(COMMON_FORMAT),
+      end: _end.subtract(diff, 'days').format(COMMON_FORMAT)
+    }
+  }
+
+  const handleChange = ({ start, end }, isComparing, compareMethod) => {
+    if (!onChange) {
+      return
+    }
+
+    if (!multiple) {
+      onChange({ start, end })
+      return
+    }
+
+    if (!isComparing) {
+      onChange([{ start, end }])
+      return
+    }
+
+    const secondInterval = computeSecondInterval({ start, end }, compareMethod)
```

Comment:
Maybe use `moment(start)` etc when passing the params here and avoid naming the variables `_start` and `_end` in the `computeSecondInterval` method?

Reply:
https://github.com/etvascom/etvas-internals/pull/41/commits/c11be831dfc1a964c8e645592d8cf14dc40d3c1c